tests/telemetry/test_writer.py

```diff
@@ -441,8 +441,8 @@ def test_app_client_configuration_changed_event(telemetry_writer, test_agent_ses
 
         telemetry_writer.periodic()
 
-        events = test_agent_session.get_events()
-        assert len(events) == initial_event_count + 1
+        events = test_agent_session.get_events("app-client-configuration-change")
+        assert len(events) >= initial_event_count + 1
```

Comment:
should `initial_event_count` be `len(test_agent_session.get_events("app-client-configuration-change")`?

would it make it more consistent then?

also, for the occasional more than 1 event, I couldn't tell... which events show up? are those both `app-client-configuration-change` events?

Reply:
I don't know, haven't verified it experimentally.